### PR TITLE
[new] add the campaigin library to the deployment process

### PR DESCRIPTION
**Pull request by GustavoRSSilva**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -26,6 +26,15 @@ IAB_KOVAN_ADDRESS=address_iab_kovan
 IAB_MAINNET_ADDRESS=0xb015D9bBabc472BBfC990ED6A0C961a90a482C57
```
Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

**Review by tcrosado — APPROVED** [review]

**Comment by GustavoRSSilva** [comment]
@tcrosado The other PR add code that belong to this one, so I merged both into one PR.

File: .env.example (hunk 1 above)
Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example:26]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

**Comment by tcrosado** [comment]
@GustavoRSSilva can you solve this conflicts to merge this pr?

**Comment by GustavoRSSilva** [comment]
@tcrosado  Going to close this PR, the new changes changed the landscape and there are now a file for each contract, making this PR deprecated.

**Pull request closed without merging**